internal/backend/service/event_test.go

```diff
@@ -14,7 +14,7 @@ const TEST_SERVER_ID = "test-server-id"
 
 func TestEventService_PublishEvent(t *testing.T) {
 	// PublishEvent doesn't use dbClient, so we can pass nil
-	service := NewEventService(nil)
+	service := NewEventService(nil, "")
```

Comment:
Surely we could have something like "test-jwt-token" as the argument. Applies to the other test files too